README.md

```diff
@@ -71,23 +110,23 @@ makepkg -si
 sudo pacman -U deltachat-desktop-git-<version>.tar.xz
 ```
 
+</details>
+
 ### Mac OS <a id="mac"></a>
 
-### Homebrew
+#### Homebrew
 
 ```
 $ brew install --cask deltachat
 ```
 
-### DMG
+#### DMG
 
 Simply install the `.dmg` file as you do it with all other software on mac.
 
-If you are getting an OpenSSL error message at the first start up you need to install OpenSSL.
+### Windows <a id="windows"></a>
 
-```
-$ brew install openssl
-```
+You can find the downloads for windows on <https://get.delta.chat>, though we recomend using the [microsoft store release](https://www.microsoft.com/en-us/p/deltachat/9pjtxx7hn3pk?activetab=pivot:overviewtab), because there you get automatic updates.
```

Comment:
```suggestion
You can find the downloads for windows on <https://get.delta.chat>, though we recomend using the [Microsoft Store release](https://www.microsoft.com/en-us/p/deltachat/9pjtxx7hn3pk?activetab=pivot:overviewtab), because there you get automatic updates.
```